umbraco/Umbraco-CMS
# Remove "(blank)" from listview create options if content has no templates

### Prerequisites


### Description

All content types are prefixed with "(blank)" in the create options in listviews, regardless if they have content templates or not. This is possibly an unintended side effect of #3173.

![content-templates-before](https://user-images.githubusercontent.com/7405322/50042537-918a4b80-0064-11e9-9cec-5312fb61faa0.png)

This PR removes the "(blank)" prefix from content types that have no content templates:

![content-templates-after](https://user-images.githubusercontent.com/7405322/50042546-b383ce00-0064-11e9-8f0c-149266e16d89.gif)

To test the PR:

1. Create a doctype that displays its children in a listview.
2. Allow the doctype as a child of itself.
3. Make sure the doctype has no content templates.
4. Verify that the doctype has no "(blank)" prefix in the listview create options.
5. Create one or more content templates for the doctype.
6. Verify that the doctype now has the "(blank)" prefix in the listview create options.
7. Verify that the content templates are available in the listview create options.

🎄 15/24 🎄
